Mathlib/LinearAlgebra/Matrix/Adjugate.lean

```diff
@@ -417,6 +417,210 @@ theorem adjugate_fin_two (A : Matrix (Fin 2) (Fin 2) α) :
 theorem adjugate_fin_two_of (a b c d : α) : adjugate !![a, b; c, d] = !![d, -b; -c, a] :=
   adjugate_fin_two _
 #align matrix.adjugate_fin_two_of Matrix.adjugate_fin_two_of
+--adjugate !![a, b, c; d, e, f; g, h, i]
```

Comment:
I don't think this is needed, since it's exactly the statement of the second lemma anyway.
```suggestion

```